R/warning-utils.R

```diff
@@ -0,0 +1,98 @@
+# Source constants
+source("R/constants.R")
+
+#' Missing warning.
+#' 
+#' Warning for when information is missing for a part.
+#' 
+#' @param column_name string storing the column name. 
```

Comment:
The parameter descriptions should be made more human readable. For example, `The name of the column with missing information`

Reply:
Can you go through all the functions and update the parameter descriptions.

Reply:
> The parameter descriptions should be made more human readable. For example, `The name of the column with missing information`

In previous PR's you have mentioned I should be including the data type of the parameter. But your current suggestion does not mention data type of the input.